swift-sdk/Internal/InAppHelper.swift

```diff
@@ -301,15 +301,15 @@ struct InAppHelper {
         case failure(reason: String, messageId: String?)
     }
     
-    /// This is a struct equivalent of IterableHtmlInAppContent class
+    /// This is a struct equivalent of IterableInAppMessage class
     private struct InAppDetails {
         let channelName: String
         let messageId: String
         let campaignId: String
         let edgeInsets: UIEdgeInsets
         let backgroundAlpha: Double
         let html: String
-        let extraInfo: [AnyHashable : Any]?
+        let extraInfo: [String : String]?
```

Comment:
This user-defined JSON payload can be nested, and if we serialize it as key-value, it might not be the same as the input when deserialized from a file.